src/test/java/seedu/address/logic/commands/TakeAttendanceCommandTest.java

```diff
@@ -90,6 +92,32 @@ public void execute_takeAttendanceWithMultipleStudents_success() throws CommandE
         assertFalse(updatedSession.getStudents().contains(bob));
     }
 
+    @Test
+    public void execute_invalidArguments_throwsCommandException() {
```

Comment:
throws `SessionNotFoundException` ? not `CommandException`

Reply:
youre right, i split the test into two as well